subtrack.MAUI/Pages/CreateSubscription.razor

```diff
@@ -1,13 +1,36 @@
 ﻿@page "/CreateSubscription"
 @inject NavigationManager NavigationManager
 
-<h3>CreateSubscription</h3>
-<br/>
-<p>returnUrl: @ReturnUrl</p>
+<div style="display: flex; justify-content: space-between; padding-bottom: 20px;">
+    <button type="button" class="btn btn-outline-dark">Back</button>
```

Comment:
Add the onclick functionality and navigate to the returnurl

Reply:
added in most recent commit